asb-ballerina/Module.md

```diff
@@ -1,473 +1,262 @@
 
-### Step 6: Initialize the Input Values
-Initialize the message to be sent with message body, optional parameters and properties.
-```ballerina
-    // Input values
-    string stringContent = "This is My Message Body"; 
-    byte[] byteContent = stringContent.toBytes();
-    map<string> properties = {a: "propertyValue1", b: "propertyValue2"};
-    int timeToLive = 60; // In seconds
-    int serverWaitTime = 60; // In seconds
-
-    asb:ApplicationProperties applicationProperties = {
-        properties: {a: "propertyValue1", b: "propertyValue2"}
-    };
+    Send a message to the Azure Service Bus
 
-    asb:Message message1 = {
-        body: byteContent,
-        contentType: asb:TEXT,
-        timeToLive: timeToLive,
-        applicationProperties: applicationProperties
-    };
-```
+    ```ballerina
+        public function main() returns error? {
+            handle queueSender = check asbClient->createQueueSender(queueName);
 
-### Step 7: Send a Message to Azure Service Bus
-You can now send a message to the configured azure service bus entity with message body, optional parameters and 
-properties. Here we have shown how to send a text message parsed to the byte array format.
-```ballerina
-    checkpanic asbClient->send(queueSender, message1);
-```
+            string stringContent = "This is My Message Body"; 
+            byte[] byteContent = stringContent.toBytes();
+            int timeToLive = 60; // In seconds
 
-### Step 8: Receive a Message from Azure Service Bus
-You can now receive a message from the configured azure service bus entity.
-```ballerina
-    asb:Message|asb:Error? messageReceived = asbClient->receive(queueReceiver, serverWaitTime);
-
-    if (messageReceived is asb:Message) {
-        log:printInfo("Reading Received Message : " + messageReceived.toString());
-    } else if (messageReceived is ()) {
-        log:printError("No message in the queue.");
-    } else {
-        log:printError("Receiving message via Asb receiver connection failed.");
-    }
-```
+            asb:ApplicationProperties applicationProperties = {
+                properties: {a: "propertyValue1", b: "propertyValue2"}
+            };
 
-### Step 9: Close Sender Connection
-You can now close the sender connection.
-```ballerina
-    checkpanic asbClient->closeSender(queueSender);
-```
+            asb:Message message = {
+                body: byteContent,
+                contentType: asb:TEXT,
+                timeToLive: timeToLive,
+                applicationProperties: applicationProperties
+            };
 
-### Step 10: Close Receiver Connection
-You can now close the receiver connection.
-```ballerina
-    checkpanic asbClient->closeReceiver(queueReceiver);
-```
+            check asbClient->send(queueSender, message);
+
+            check asbClient->closeSender(queueSender);
+        }
+    ```
+
+    Following is an example on how to receive messages from the Azure Service Bus using the client connector. You need to create a 
```

Comment:
remove unnecessary spaces.

Reply:
This sentence is indented for a proper view. It's not an unnecessary space.